src/UI/templates/js/Input/Field/tagInput.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Wraps the BootstrapTagsInput
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+var il = il || {};
+il.UI = il.UI || {};
+il.UI.Input = il.UI.Input || {};
+(function ($, UI) {
```

Comment:
This is supposed to introduce a local scope to prevent functions defined in it from leaking into the global scope, right? Why do you pass UI, then, without using it?

Reply:
Also, there doesn't seem to be a thing defined in this scope. Or is there anything I'm missing about this construct?